gateway/package.json

```diff
@@ -18,11 +17,17 @@
   },
   "scripts": {
     "dev": "NODE_ENV=development IS_DEV=true VARIANT=default nodemon",
-    "dev:admin": "NODE_ENV=development IS_DEV=true VARIANT=admin nodemon"
+    "dev:admin": "NODE_ENV=development IS_DEV=true VARIANT=admin nodemon",
```

Comment:
why do we need both node_env and is_dev?

Reply:
🤔 I assume to allow to start with production bundle without https

Reply:
I will change to use NODE_ENV for now then we will see